lib/buffer.txt

```diff
@@ -0,0 +1,216 @@
+class PersonalForm extends StatefulWidget {
+  const PersonalForm({super.key});
+
+  @override
+  State<PersonalForm> createState() => _PersonalFormState();
+}
+
+class _PersonalFormState extends State<PersonalForm> {
+  @override
+  void initState() {
+    super.initState();
+
+    _firstNameController = TextEditingController();
+    _lastNameController = TextEditingController();
+  }
+
+  @override
+  void dispose() {
+    _firstNameController.dispose();
+    _lastNameController.dispose();
+
+    super.dispose();
+  }
+
+  late TextEditingController _firstNameController;
+  late TextEditingController _lastNameController;
+
+  String _emailError = '';
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        iconTheme: const IconThemeData(color: primaryColor),
+      ),
+      body: Padding(
+        padding: const EdgeInsets.all(32),
+        child: SingleChildScrollView(
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.stretch,
+            children: [
+              Image.asset('assets/logo.png'),
+              const Text(
+                'Enter Personal Details',
+                textAlign: TextAlign.center,
+                style: TextStyle(
+                  color: primaryColor,
+                  fontSize: 26,
+                  fontWeight: FontWeight.w600,
+                  height: 1.5,
+                ),
+              ),
+              const Text(
+                'Enter personal information below',
+                style: TextStyle(color: primaryColor, fontSize: 16),
+              ),
+              const SizedBox(height: 32),
+              FormBox(
+                icon: Icons.email_outlined,
+                hintText: 'Email',
+                controller: _firstNameController,
+              ),
+              if (_emailError.isNotEmpty) ...[
+                const SizedBox(height: 16),
+                Text(
+                  _emailError,
+                  style: const TextStyle(color: Colors.red, fontSize: 16),
+                ),
+              ],
+              const SizedBox(height: 16),
+              FormBox(
+                icon: Icons.lock_outline_rounded,
+                hintText: 'Password',
+                obscureText: true,
+                controller: _lastNameController,
+              ),
+              const SizedBox(height: 16),
+//              FormBox(
```

Comment:
why are these commented out?  If they should not be there, I would go ahead and remove them.